sdk/core/Azure.Core/feature_lifecycle.md

```diff
@@ -0,0 +1,31 @@
+# Core Ship Sequence (draft)
+Legend:
+- diamond shapes == Phase
+- hexagonal shapes == Sub Step
+
+```mermaid
+flowchart TD
+    A{Feature Definition} --> B{{Feature Triage}}
+    B -->|Committed| SelS{{Select Shipping Sprint}}
+    SelS --> AtS{{Add Issue to Sprint Plan}}
+    AtS --> DI{Design and<Br/> Implementation}
+    DI --> DDes{{Document Design}}
+    DDes --> Cost{{Costing of Feature}}
+    Cost --> DDec{{Document Decisions}}
+    DDec --> AP{Approval Phase}
+    AP --> Arch{{Arch Board}}
+    Arch --> DocCh{{Document any Changes}}
+    DocCh --> ApplyFB{{Apply Feedback}}
+    ApplyFB --> Merge{{Merge to Main}}
+    Merge --> FinalApp{{Final APIView Approval}}
```

Comment:
Since in real life there is always more work that arises at the APIView approval stage, we might want to represent that here.